### PR TITLE
Install sentry

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,10 +4,31 @@ import './wdyr';
...
 
+const history = createBrowserHistory();
+
+const { REACT_APP_SENTRY_DSN, REACT_APP_SENTRY_ENVIRONMENT, REACT_APP_VERSION } = process.env;
+
+if (REACT_APP_SENTRY_DSN !== undefined) {
+  Sentry.init({
+    dsn: REACT_APP_SENTRY_DSN,
+    ...(REACT_APP_SENTRY_ENVIRONMENT && { environment: REACT_APP_SENTRY_ENVIRONMENT }),
+    ...(REACT_APP_VERSION && { release: `GeoCRUD-front@${REACT_APP_VERSION}` }),
```
Comment (submarcos) [inline src/index.js:23]: I suggest to include sample Rate management https://docs.sentry.io/platforms/javascript/configuration/sampling/

**Review by submarcos — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (dtrucs) [reply src/index.js:23]: Oh yes, top!
Do you have a suggestion of the best value to take. I don't have enough experience in Sentry to choose one. By default I will copy/paste the example into the doc.

Comment (submarcos) [reply src/index.js:23]: it's a range from **0.0** to **1.0**.  0 is disabled, so i suggest to use 0 by default

Comment (dtrucs) [reply src/index.js:23]: I'm not sure about this value 

> to a number between 0 (0% of errors sent) and 1 (100% of errors sent)

Sentry should become useless if it no longer sends errors?

File: src/index.js [hunk 2]
```diff
@@ -4,10 +4,31 @@ import './wdyr';
 
 import React from 'react';
 import ReactDOM from 'react-dom';
+import * as Sentry from '@sentry/react';
```
Comment (jrmi) [inline src/index.js:7]: Can you make import only if Sentry is used (i.e. REACT_APP_SENTRY_DSN var is not undefined)

Comment (dtrucs) [reply src/index.js:7]: wow i didn't know it was possible, i try it right now 🙏

File: src/index.js (hunk 1 above)
Comment (submarcos) [reply src/index.js:23]: SampleRate is not for error reporting, it's for performance monitoring. 0 -> 0% of transactions are monitored. 1 -> 100% of transactions are monitored. In all case, 0 to 1, errors are reported

Comment (submarcos) [reply src/index.js:23]: Oups ! it's not sampleRate, it's tracesSampleRate

Comment (submarcos) [reply src/index.js:23]: sampleRate is for error, so 1 by default. tracesSampleRate is for performance monitoring, so 0 by default

Comment (dtrucs) [reply src/index.js]: Done ! 👍

File: src/index.js (hunk 2 above)
Comment (dtrucs) [reply src/index.js]: Done 👍

**Review by jrmi — APPROVED** [review]

**Pull request merged**